test/spacesuit_auth_middleware_test.exs

```diff
@@ -67,5 +67,27 @@ defmodule SpacesuitAuthMiddlewareTest do
       # Unrecognized, we pass it on as is
       assert {:ok, ^req, ^env} = Spacesuit.AuthMiddleware.execute(req, env)
     end
+
+    test "with a valid token on a bypassed path does not invoke session service" do
+      # omit a session service since it will *not* be invoked
+      Application.put_env(:spacesuit, :session_service, %{ enabled: true, bypassed_sigils: [~r{/skipme}] })
+
+      req = %{ headers: %{ "authorization" => "Bearer " }, pid: self(), streamid: 1, method: "GET", path: "/skipme" }
+      env = %{}
+
+      # pass it on as is
+      assert {:ok, ^req, ^env} = Spacesuit.AuthMiddleware.execute(req, env)
+    end
+
+    assert_raise UndefinedFunctionError, ~r{^function nil.handle_bearer_token/4 is undefined or private}, fn ->
+      # omit a session service to trigger error
+      Application.put_env(:spacesuit, :session_service, %{ enabled: true })
+
+      req = %{ headers: %{ "authorization" => "Bearer " }, pid: self(), streamid: 1, method: "GET", path: "/skipme" }
+      env = %{}
+
+      # trigger the error
+      Spacesuit.AuthMiddleware.execute(req, env)
+    end
```

Comment:
on 2nd thought, I'm not sure this test case makes sense. was trying to figure out a way to assert that the session service was in fact bypassed when the sigil matched but that may already be expressed in the fact that its omitted in the env above so this test is possibly redundant.